### PR TITLE
Bug 1999186: Add MCO and CVO related config excludes

### PR DESCRIPTION
**Pull request by mrogers950**
@JAORMX

### CODE REVIEW

File: pkg/controller/fileintegrity/config_defaults.go [hunk 1]
```diff
@@ -45,6 +45,9 @@ CONTENT_EX = sha512+ftype+p+u+g+n+acl+selinux+xattrs
 !/hostroot/etc/selinux/targeted
 !/hostroot/etc/openvswitch/conf.db
 !/hostroot/etc/kubernetes/cni/net.d/*
+!/hostroot/etc/machine-config-daemon
+!/hostroot/etc/pki/ca-trust/extracted/java
+!/hostroot/etc/cvo
```
Comment (JAORMX) [inline pkg/controller/fileintegrity/config_defaults.go:50]: I don't think we should fully exclude these, but at least we should be checking that the appropriate permissions are kept.

File: pkg/controller/fileintegrity/config_defaults.go [hunk 2]
```diff
@@ -45,6 +45,9 @@ CONTENT_EX = sha512+ftype+p+u+g+n+acl+selinux+xattrs
 !/hostroot/etc/selinux/targeted
 !/hostroot/etc/openvswitch/conf.db
 !/hostroot/etc/kubernetes/cni/net.d/*
+!/hostroot/etc/machine-config-daemon
+!/hostroot/etc/pki/ca-trust/extracted/java
```
Comment (JAORMX) [inline pkg/controller/fileintegrity/config_defaults.go:49]: isn't this merely a PKCS12 file? that one shouldn't change as often.

**Review by JAORMX — CHANGES_REQUESTED** [review]

File: pkg/controller/fileintegrity/config_defaults.go (hunk 1 above)
Comment (mrogers950) [reply pkg/controller/fileintegrity/config_defaults.go:50]: Actually I think there's a regex pattern I can use to still watch for weird stuff in the directory and omit the dynamic files. Gotta look into it some more.

**Comment by mrogers950** [comment]
@JAORMX I've updated it to a more specific exclude, so we should have better coverage with this iteration.

**Pull request merged**